### ISSUE TITLE
checklist don't display "checked" attribute

### ISSUE THREAD

**Issue by manuel-84** [vitalets/x-editable-yii#24]
``` js
//set checked on required checkboxes
       value2input: function(value) {
```

value is null when only 1 element should be selected, instead it is the correct array when we have 2 or more elements selected

the problem seems to be the str2value function: it doesnt handle the case when str is not a string or array and returns null

``` js
        str2value: function(str) {
           var reg, value = null;
           if(typeof str === 'string' && str.length) {
               reg = new RegExp('\\s*'+$.trim(this.options.separator)+'\\s*');
               value = str.split(reg);
           } else if($.isArray(str)) {
               value = str; 
           }
           return value;
        },   
```

i edited the function adding an else and now it-s working

``` js
        str2value: function(str) {
           var reg, value = null;
           if(typeof str === 'string' && str.length) {
               reg = new RegExp('\\s*'+$.trim(this.options.separator)+'\\s*');
               value = str.split(reg);
           } else if($.isArray(str)) {
               value = str;
           } else {
             value = new Array;
             value.push(str);
           }
           return value;
        },
```

reference> line 2357 of bootstrap-editable-inline.js ... but also the other js

**Comment by vitalets** [comment]
ok, I'will add this to original x-editable lib 1.4.4.
thanks a lot!

**Comment by vitalets** [comment]
fixed in 1.1.0
